### PR TITLE
changing type of java docs to symlinks

### PR DESCRIPTION
**Pull request by arpanchaudhury**
1. java/stream/stages-overview.md -> scala/stream/stages-overview.md
2. java/stream/stream-composition.md -> scala/stream/stream-composition.md
3. java/stream/stream-flows-and-basics.md -> scala/stream/stream-flows-and-basics.md

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

File: akka-docs/src/main/paradox/java/stream/stages-overview.md [hunk 1]
```diff
@@ -1 +0,0 @@
-../../scala/stream/stages-overview.md
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/stream/stages-overview.md]: These should be relative links - they don't point anywhere now.

**Review by raboof — CHANGES_REQUESTED** [review]

File: akka-docs/src/main/paradox/java/stream/stages-overview.md (hunk 1 above)
Comment (arpanchaudhury) [inline akka-docs/src/main/paradox/java/stream/stages-overview.md]: I just created symlinks e.g. ```ln -s akka-docs/src/main/paradox/scala/stream/stream-composition.md akka-docs/src/main/paradox/java/stream/stream-composition.md```

Comment (raboof) [inline akka-docs/src/main/paradox/java/stream/stages-overview.md]: this doesn't work, as symlinks are relative so now "akka-docs/src/main/paradox/java/stream/stream-composition.md" is a symlink to "akka-docs/src/main/paradox/java/stream/akka-docs/src/main/paradox/scala/stream/stream-composition.md" which obviously doesn't exist :)

you can test symlinks simply by looking at the contents, i.e. "cat akka-docs/src/main/paradox/java/stream/stream-composition.md"

Comment (arpanchaudhury) [inline akka-docs/src/main/paradox/java/stream/stages-overview.md]: Ah. I see. Thanks for the explanation. I will change it. 😃

**Comment by arpanchaudhury** [comment]
@raboof PR Updated

**Comment by raboof** [comment]
Good catch thanks!

**Pull request merged**